src/solve_quadratic.cpp

```diff
@@ -188,25 +196,47 @@ void WriteAnswer (int nRoots, double x1, double x2) {
             printf ("main(): ERROR: nRooTs = %d\n", nRoots);
     }
 }
+//--------------------------------------------------------------------------//
+
+//--------------------------------------------------------------------------//
+/*! \fn void EnterToTestSolver
+ *  \brief Invite you to check the program on unit tests
+ */
+//--------------------------------------------------------------------------//
+void EnterToTestSolver () {
```

Comment:
Use either a conditional compiling for testing or process command-line arguments and if there's flag "-t" or "--test", then run tests.